### DISCUSSION TITLE
Preferred Vivado version for FINN

### DISCUSSION THREAD

**Discussion by sakshigoel361** [Xilinx/finn#440] [Q&A] [answered]
Hello everyone,
I am a new user looking into FINN and wanting to a project on it. I was trying the 3rd notebook in end_2_end_examples/cybersecurity and I get the error "AssertionError: vivado hls not found in PATH"
Now I have set the path for Vivado to latest vivado installation 2021.1. 
gsakshi: FINN/finn$ which vivado
/mnt/shareLin/Xilinx/Vivado/2021.1/bin/vivado
Is this the correct version compatible with finn-hls or I should use 2019.1? I saw in an old post from Feb2020 which suggested using 2019.1- Also I have Vitis_HLS installed and not Vivado_HLS. Should I chose Vivado and not VitisAI while doing the installation?
Thank you for any answers. Looking forward to learning new things with this project

**Comment by fpjentzsch** [comment, marked as answer]
Hi,
Vivado HLS is no longer available in 2021.1 and Vitis HLS is not fully supported yet, so I suggest to use Vivado/Vivado HLS 2020.1.

2019.1 should also work.
